cohere/client.py

```diff
@@ -403,12 +403,14 @@ def feedback(self, id: str, good_response: bool, desired_response: str = "", fee
     def rerank(self,
                query: str,
                documents: Union[List[str], List[Dict[str, Any]]],
+               model: str = None,
                top_n: Optional[int] = None) -> Reranking:
         """Returns an ordered list of documents ordered by their relevance to the provided query
 
         Args:
             query (str): The search query
             documents (list[str], list[dict]): The documents to rerank
+            model (str): (Optional) The model to use for re-ranking
```

Comment:
could we say what the defaults are for these args in the doc string?

Reply:
the default is none

Reply:
i mean the default behavior, if it's none we default to the english model right?

Reply:
yeah but the default model could change, it will be set in the docs once this goes public